src/app/component/ExpandableContent/ExpandableContent.component.js

```diff
@@ -66,10 +73,14 @@ export class ExpandableContent extends PureComponent {
     }
 
     toggleExpand = () => {
-        const { onClick } = this.props;
+        const { onClick, productReviewsRef, scrollToPosition } = this.props;
+        const { isContentExpanded } = this.state;
         if (onClick) {
             onClick(); return;
         }
+
+        scrollToPosition(productReviewsRef, isContentExpanded);
```

Comment:
`ExpandableContent` logic is NOT related to `ProductReviews` logic!

Consider adding an option in the `ExpandableContent` component which is going to scroll to the shown element (maybe with additional argument to subtract the covering elements).

Try encapsulating the scrolling functionality in the component which utilises it!